# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mine/net-install.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Of course we do not want curl in one place and wget at the other, also
there is no wget in the second stange anymore ...

  - Rene Rebe <rene@exactcode.de>

diff -ur mine-0.23/readdb.c mine-0.23-fixed/readdb.c
--- mine-0.23/readdb.c	2006-06-30 18:20:36.000000000 +0200
+++ mine-0.23-fixed/readdb.c	2006-06-30 18:10:59.000000000 +0200
@@ -441,7 +441,7 @@
 		snprintf(filename, PATH_MAX, "gzip -d < %s/%s/pkgs/packages.db",
 		         sourcedir, config);
 	else
-		snprintf(filename, PATH_MAX, "wget -O - -nv %s/%s/pkgs/packages.db | gzip -d",
+		snprintf(filename, PATH_MAX, "curl -s -S %s/%s/pkgs/packages.db | gzip -d",
 		         sourcedir, config);
 	if ( (f = popen(filename, "r")) != NULL ) {
 		char *package, line[160];
